### ISSUE TITLE
Exclude page routes but keep dynamic endpoint routes

### ISSUE THREAD

**Issue by DarkHiito** [harlan-zw/nuxt-simple-sitemap#139]
### Details

Hi guys,

I've got an endpoint on my API which returns me all the URLs that I want to display in my sitemap, so I've setted up my config like bellow :
```
sitemap: {
    autoI18n: false,
    inferStaticPagesAsRoutes: false,
    sitemaps: {
        fr: { dynamicUrlsApiEndpoint: `http://api.${process.env.BASE_DOMAIN}.fr/v1/sitemap` },
        eu: { dynamicUrlsApiEndpoint: `http://api.${process.env.BASE_DOMAIN}.eu/v1/sitemap` },
    }
}
```

The issue is that I don't want the sitemap to display any of the Nuxt pages, I only want those ones from my API.
I've tried to add `include: []` or `exclude: ['/**']` but it also excludes the routes from my API.

Can you advise me how can I managed to do what I want ?

Thank you

**Comment by harlan-zw** [comment]
Hi @DarkHiito, thanks for the issue.

You can try enabling the `debug` mode and visiting `/api/__sitemap__/debug`, this should show you where it's pulling the URLs from. If you send me the output and what you're trying to remove, I can guide you on how to remove it.

**Comment by DarkHiito** [comment]
@harlan-zw Up ? 😅

**Comment by harlan-zw** [comment]
Hey, thanks for the bump.

This should be fixed in 3.3.5. The URLs seem to have been coming from the `i18n.pages` config, this will now respect the `autoI18n` config.

**Comment by DarkHiito** [comment]
It's working well with 3.3.5, thanks !
